crypto/tmhash/bench_test.go

```diff
@@ -1,52 +0,0 @@
-package tmhash
-
-import (
-	"bytes"
-	"crypto/sha256"
-	"strings"
-	"testing"
-)
-
-var sink any
-
-var manySlices = []struct {
-	name string
-	in   [][]byte
-	want [32]byte
-}{
-	{
-		name: "all empty",
-		in:   [][]byte{[]byte(""), []byte("")},
-		want: sha256.Sum256(nil),
-	},
-	{
-		name: "ax6",
-		in:   [][]byte{[]byte("aaaa"), []byte("😎"), []byte("aaaa")},
-		want: sha256.Sum256([]byte("aaaa😎aaaa")),
-	},
-	{
-		name: "composite joined",
-		in:   [][]byte{bytes.Repeat([]byte("a"), 1<<10), []byte("AA"), bytes.Repeat([]byte("z"), 100)},
-		want: sha256.Sum256([]byte(strings.Repeat("a", 1<<10) + "AA" + strings.Repeat("z", 100))),
-	},
-}
-
-func BenchmarkSHA256Many(b *testing.B) {
```

Comment:
Not that I'm against removing code (I actually love the idea 😄), but what is the reason for deleting this benchmark?